Formula/k/kubeshark.rb

```diff
@@ -0,0 +1,28 @@
+class Kubeshark < Formula
+  desc "API Traffic Analyzer providing real-time visibility into Kubernetes network"
+  homepage "https://github.com/kubeshark/kubeshark"
+  url "https://github.com/kubeshark/kubeshark.git",
+    tag:      "v52.1.63",
+    revision: "68aabf262fdf6f3fe8f4aaef089f6f7c2471656e"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    args = %W[
+      -s -w
+      -X "github.com/kubeshark/kubeshark/misc.Platform=#{OS.kernel_name}_#{Hardware::CPU.arch}"
+      -X "github.com/kubeshark/kubeshark/misc.BuildTimestamp=#{time}"
+      -X "github.com/kubeshark/kubeshark/misc.Ver=v#{version}"
+    ]
+    system "go", "build", *std_go_args(ldflags: args.join(" "))
```

Comment:
consistent with other go formulae
```suggestion
    ldflags = %W[
      -s -w
      -X "github.com/kubeshark/kubeshark/misc.Platform=#{OS.kernel_name}_#{Hardware::CPU.arch}"
      -X "github.com/kubeshark/kubeshark/misc.BuildTimestamp=#{time}"
      -X "github.com/kubeshark/kubeshark/misc.Ver=v#{version}"
    ]
    system "go", "build", *std_go_args(ldflags: ldflags)
```